.eslintrc.json

```diff
@@ -3,14 +3,25 @@
     "browser": true,
     "es2021": true
   },
-  "extends": ["next","airbnb", "prettier"],
+  "extends": ["next", "airbnb", "prettier"],
   "plugins": ["react", "prettier"],
   "parser": "@typescript-eslint/parser",
   "rules": {
     "prettier/prettier": ["error"],
     "react/jsx-filename-extension": [2, { "extensions": [".tsx"] }],
     "react/jsx-props-no-spreading": "off",
     "react/prop-types": "off",
-    "react/react-in-jsx-scope": "off"
+    "react/react-in-jsx-scope": "off",
+    
+    "import/extensions": [
+      "error",
+      "ignorePackages",
+      {
+        "js": "never",
+        "jsx": "never",
+        "ts": "never",
+        "tsx": "never"
+      }
+    ]
```

Comment:
❓ 
why do you still have this? just want to know what does it do?

Reply:
I forgot to answer this. Sorry, my bad.
**Missing file extension "ts"** When I import any component  this error appears to me and when I searched about the rezone I found that I am on the wrong path  or **ts** is missing but I am in the right path also the extension not missing.